Automattic/sensei
# Fix/issue 2826 Filter Prerequisite dropdown

Fixes #2826 

#### Changes proposed in this Pull Request:
* This PR modifies the query used to get lessons for the prerequisite dropdown.

#### Testing instructions:
* Go to the Add/Edit Lesson page
* If the lesson is attached to a course, the prerequisite dropdown should show only lessons from that course, except for the lesson that is currently being edited
* If the lesson is not attached to a course, the prerequisite dropdown should not be displayed

#### Proposed changelog entry for your changes:
* Filter prerequisite dropdown in Add/Edit Lesson page to only show lessons from the current lesson's course
